community/releases/support-policy.md

```diff
@@ -10,19 +10,53 @@ redirect_from: /docs/support-policy
 
 ## Bugs and Feature Requests
 
-If you have what looks like a bug, or you would like to make a feature request, please use the [GitHub issue tracking system](https://github.com/spinnaker/spinnaker/issues).
+If you have what looks like a bug, or you would like to make a feature request,
+please use the [GitHub issue tracking
+system](https://github.com/spinnaker/spinnaker/issues).
 
-Before you file an issue, please search existing issues to see if your issue is already covered.
+Some things to keep in mind:
+
+* Please search existing issues to see if your issue is already covered. If it
+  is, and you have any new information, please share it! It will help us close
```

Comment:
s/share/add

I doubt anyone would misunderstand and create a new issue, but you never know.